### ISSUE TITLE
Doesn't catch extra space before function call parens

### ISSUE THREAD

**Issue by hadley** [jimhester/lintr#219]
e.g.

```R
plot (x, y)
```

**Comment by jimhester** [comment]
Fixed by ec6692bec3053a75c785b426ad81ecadebf3de5d, b0d2b4be7ae371221a70b39406744db0af766b48
